database.py

```diff
@@ -0,0 +1,53 @@
+import sqlite3
+import datetime
+# Create / connect to database
+con = sqlite3.connect("timelord.db")
+cur = con.cursor()
+cur.execute("CREATE TABLE IF NOT EXISTS timeLog(userID TEXT NOT NULL, selectedDate date, minutes INTEGER NOT NULL)")
+con.close()
+
+today = datetime.date.today()
```

Comment:
A new SQL Connection object is created and destroyed for each slash command being executed because:

1. The time between commands is unknown and the database is only saved to disk when the connection is closed.
2. The Slack Bolt SDK uses multithreading. If the connection was opened when database.py was imported it couldn't be used in any of the listener functions unless we serialize write operations - https://docs.python.org/3/library/sqlite3.html#module-functions

The table only needs to be checked for once when the bot starts up, not each time a new query is made

Reply:
Gotcha. I would be inclined to have this be a function that can be called from `app.py` rather than having global code in a class file